fairroot/base/steer/FairRunAna.h

```diff
@@ -35,25 +35,34 @@ class FairRunAna : public FairRun
     FairRunAna();
     /**initialize the run manager*/
     void Init() override;
-    /**Run from event number NStart to event number NStop */
-    void Run(Int_t NStart = 0, Int_t NStop = 0) override;
+    /**Run analysis:
+     * for the whole input if NEntry == NStop == 0,
+     * for NEntry entries if NStop == 0,
+     * from NEntry until NStop entry is reached
+     * from NEntry to the end of input, if NStop = -1 */
+    void Run(FairRoot::EntryID NEntry = 0, int NStop = 0) override;
     /**Run over the whole input file with timpe window delta_t as unit (entry)*/
     void Run(Double_t delta_t);
+    /** \deprecated Deprecated in v19, will be removed in v20. */
+    [[deprecated("Replaced by RunSingle(), which has an unambiguous name")]] void Run(Long64_t entry)
+    {
+        RunSingle(entry);
+    }
     /**Run for the given single entry*/
-    void Run(Long64_t entry);
-    /**Run event reconstruction from event number NStart to event number NStop */
-    void RunEventReco(Int_t NStart, Int_t NStop);
```

Comment:
we also should deprecated the int, int overloads first before removing them, right?